src/components/Modals/EditExpenseModal.js

```diff
@@ -32,14 +26,10 @@ const EditExpenseModal = forwardRef((props, ref) => {
     e.preventDefault();
     setShowModal(false);
 
-    const newExpenseData = {
-      title,
-      date: date.toString(),
-      amount: convertEurosToCents(amount),
-    };
+    const newExpenseData = { title, date, amount };
 
     try {
-      await editExpense(id, newExpenseData);
+      expense.update(newExpenseData);
```

Comment:
question: no need to `await` here any more?

Reply:
Good catch!